frontend/src/components/EnvironmentConfiguration.jsx

```diff
@@ -485,6 +506,78 @@ export default function EnvironmentConfiguration (env) {
                     height: '1em',
                 }
         }));
+
+        // SADE AREA //
+        const classes = useStyles();
+
+        const [sadeCount, setSadeCount] = React.useState(env.mainJsonValue.Sades != null ? env.mainJsonValue.Sades.length: 1);
+        const [sadeArray, setSadeArray] = React.useState(env.mainJsonValue.Sades != null ? env.mainJsonValue.Sades : [{
+            id:sadeCount-1,
+            sadeName: "SADE " + sadeCount,
+            Name:"SADE " + (sadeCount)
+        }]);
+        
+        const setSade = () => {
+            sadeArray.push({
+                id: (sadeCount),
+                sadeName:"SADE " + (sadeCount+1),
+                Name:"SADE " + (sadeCount+1)
+            })
+        }
+
+        const popSade = () =>{
+            sadeArray.pop()
+        }
+        
+        const handleIncrement = () => {
+            setSadeCount(sadeCount +1)
```

Comment:
@Hoshi-Okami It seems increment button has some issue. When I clicked on increment  button once, multiple Sade zones added. You can look at [this demo](https://ooo.mmhmm.app/z_YCg9Rb1FL3ck8RkibbQp)
Make sure to pull latest code from frontend-updates, drone screen has been updated in that.